packages/models/src/lib/table.unit.test.ts

```diff
@@ -33,37 +33,37 @@ describe('tableSchema', () => {
     );
   });
 
-  it('should parse table with rows and headings', () => {
+  it('should parse table with rows and headings with keys only', () => {
     const table: Table = {
       rows: [{ metrics: 'TTFB' }],
-      headings: [{ key: 'metrics', label: 'Metrics Name' }],
+      headings: [{ key: 'metrics' }],
     };
     expect(() => tableSchema().parse(table)).not.toThrow();
   });
 
-  it('should parse table with rows and headings with keys only', () => {
+  it('should parse table with rows and headings', () => {
     const table: Table = {
       rows: [{ metrics: 'TTFB' }],
-      headings: [{ key: 'metrics' }],
+      headings: [{ key: 'metrics', label: 'Metrics Name' }],
     };
     expect(() => tableSchema().parse(table)).not.toThrow();
   });
 
   it('should parse table with rows and headings and alignments', () => {
     const table: Table = {
       rows: [{ metrics: 'TTFB' }],
-      headings: [{ key: 'metrics', label: 'Metrics Name' }],
-      alignment: ['l'],
+      headings: [{ key: 'metrics', label: 'Metrics Name', align: 'l' }],
     };
     expect(() => tableSchema().parse(table)).not.toThrow();
   });
 
   it('should parse complete table', () => {
     const fullTable: Table = {
       headings: [
+        // center oftern is the default when rendering in MD or HTML
```

Comment:
Is this what you meant?

```suggestion
        // center is often the default when rendering in MD or HTML
```